api/deviceflow.go

```diff
@@ -6,6 +6,12 @@ type ApproveDeviceFlowRequest struct {
 	UserCode string `json:"userCode" example:"BDSD-HQMK"`
 }
 
+func (adfr *ApproveDeviceFlowRequest) ValidationRules() []validate.ValidationRule {
```

Comment:
This is a minor style suggestion. Moved this up from below so that the method follows immediately after the type, and before other types. This keeps the type and its methods together.